cypress/e2e/ephemeral/ee.cy.ts

```diff
@@ -0,0 +1,10 @@
+describe('Ephemeral test', () => {
+  beforeEach(() => {
+    // cy.login();
+  });
+  it('should find all services page', () => {
+    cy.visit('/allservices');
+    cy.contains('All Services').should('exist');
+    cy.get('.pf-v6-c-masthead__brand > div > .pf-v6-c-menu-toggle > .pf-v6-c-menu-toggle__text').click();
```

Comment:
I'm assuming this works, but is there a way we can avoid using selectors? Not a blocker for this PR, but if an OUIA ID can avoid using selectors, I'd favor that.

Reply:
Same here, this file is not supposed to be here.